module3/src/com/epam/arrayofcharacters/BusinessWorker.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.arrayofcharacters;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class BusinessWorker {
+    //task 1
+    public static String[] convertTitlesToSnakeCase(String[] camelCase) {
+        String[] snakeCase = new String[camelCase.length];
+        for (int i = 0; i < snakeCase.length; i++) {
+            StringBuilder stringBuilder = new StringBuilder();
+            for (int j = 0; j < camelCase[i].length(); j++) {
+                char letter = camelCase[i].toCharArray()[j];
+                if (Character.isLowerCase(letter)) {
+                    stringBuilder.append(letter);
+                } else {
+                    stringBuilder.append("_");
+                    stringBuilder.append(Character.toLowerCase(letter));
+                }
+            }
+            snakeCase[i] = stringBuilder.toString();
+        }
+        return snakeCase;
+    }
+
+    //task 2
+    public static String replaceTheWords(String text) {
+        String oldWord = "world";
+        String newWord = "letter";
+        return text.replace(oldWord, newWord);
+    }
+    //task 3
+    public static int countDigits(String text) {
+        int count = 0;
```

Comment:
Here you can convert text to chars and work with it as char array:
```suggestion
    public static int countDigits(String text) {
        final char[] characters = text.toCharArray();
        int count = 0;
```